hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/http/BaseHttpServer.java

```diff
@@ -178,11 +178,13 @@ public BaseHttpServer(MutableConfigurationSource conf, String name)
       }
 
       String baseDir = conf.get(OzoneConfigKeys.OZONE_HTTP_BASEDIR);
-      if (!StringUtils.isEmpty(baseDir)) {
-        createDir(baseDir);
-        httpServer.getWebAppContext().setAttribute(JETTY_BASETMPDIR, baseDir);
-        LOG.info("HTTP server of {} uses base directory {}", name, baseDir);
+
+      if (StringUtils.isEmpty(baseDir)) {
+        baseDir = conf.get(OzoneConfigKeys.OZONE_METADATA_DIRS) + "/webserver";
```

Comment:
@ArafatKhan2198 , Instead of relying on "ozone.metadata.dirs" config, we can have separate config for httpserver.

Reply:
@ChenSammi  suggested that if the ozone.http.basedir property is not defined, we should fall back to using `ozone.metadata.dirs`. Falling back to "ozone.metadata.dirs" is a common practice in Ozone when a more specific property is not set by the user.

Reply:
Sorry @ArafatKhan2198 ,  just missed one comment here.  Please use getOzoneMetaDirPath#ServerUtils to get the metadata dir.

Reply:
Thanks @ChenSammi , I have made the changes !!